nntrainer/layers/rnn.cpp

```diff
@@ -95,7 +96,38 @@ void RNNLayer::setProperty(const PropertyType type, const std::string &value) {
   }
 }
 
-void RNNLayer::forwarding(bool training) {}
+void RNNLayer::forwarding(bool training) {
```

Comment:
As far as I understand, RNN retains the (last) previous output and use it when inference mode. I wonder if this behavior is considered

Reply:
could you point out more specific? previous output??

Reply:
We discuss offline. I will update.